src/components/contacts/ContactAvatar.js

```diff
@@ -88,26 +61,33 @@ const sizeConfigs = colors => ({
   },
 });
 
-const ContactAvatar = ({ color, size = 'medium', value, ...props }) => {
+const ContactAvatar = ({
+  address,
+  color,
+  size = 'medium',
+  emoji,
+  ...props
+}) => {
   const { colors } = useTheme();
+  const { rainbowProfile } = useRainbowProfile(color && emoji ? '' : address);
```

Comment:
i see we're using `useQuery` in this hook, could we pass a config flag `enabled` in case we want to enable this or not? and we can just pass the address as the first param for the hook

Reply:
i believe you can still pass { enabled: false/true } as one of the parameters. you don't see it used anywhere because inside `useRainbowProfile` i have the query automatically disable if the address argument is not a valid eth address. the reason why i conditionally pass an empty string here is because `useRainbowProfile` is expecting an argument of type string, so i need to make sure i'm not passing in something null/undefined. i think it's probably a good idea to change the parameter type to `string | null | undefined` so i don't have to do this

Reply:
ah okok, it also came to my atention the fact that you're checking `color && emoji` which is not strictly related to `address`

Reply:
coming back to this comment, `i think it's probably a good idea to change the parameter type to string | null | undefined so i don't have to do this` is probably a good idea to follow the standard we've been following for use query so far

Reply:
seems like you're doing it here too https://github.com/rainbow-me/rainbow/pull/3464/files#diff-bbf66eed267847cdbcf8be45cdf7113468ab41af09880959438ee70ea6d19afdR88